File: app/src/main/java/java/android/notes/Time.java
package java.android.notes;

import java.text.SimpleDateFormat;
import java.util.Calendar;

public class Time {
    public static Calendar calendar = Calendar.getInstance();
    private static SimpleDateFormat dateFormat = new SimpleDateFormat("dd.MMMM.yyyy");

    public static void setDay   (int day)   { calendar.set(Calendar.DAY_OF_MONTH,day); }
    public static void setMonth (int month) { calendar.set(Calendar.MONTH,month);      }
    public static void setYear  (int year)  { calendar.set(Calendar.YEAR,year);        }

    public static int getDay(){ return calendar.get(Calendar.DAY_OF_MONTH); }
    public static int getMonth(){ return calendar.get(Calendar.MONTH); }
    public static int getYear(){ return calendar.get(Calendar.YEAR); }

    public static String getDateYMD(){
        return dateFormat.format(calendar.getTime());
    }

}

File: app/src/main/java/java/android/notes/Time.kt
package java.android.notes

import java.text.SimpleDateFormat
import java.util.*

object Time {
    var calendar = Calendar.getInstance()
    private val dateFormat = SimpleDateFormat("dd.MMMM.yyyy")
    var day: Int
        get() = calendar[Calendar.DAY_OF_MONTH]
        set(day) {
            calendar[Calendar.DAY_OF_MONTH] = day
        }
    var month: Int
        get() = calendar[Calendar.MONTH]
        set(month) {
            calendar[Calendar.MONTH] = month
        }
    var year: Int
        get() = calendar[Calendar.YEAR]
        set(year) {
            calendar[Calendar.YEAR] = year
        }
    val dateYMD: String
        get() = dateFormat.format(calendar.time)
}